android-client-sdk/src/main/java/com/devcycle/sdk/android/model/PopulatedUser.kt

```diff
@@ -48,7 +48,7 @@ internal data class PopulatedUser constructor(
     @Schema(description = "DevCycle SDK type")
     val sdkType: String = "mobile",
     @Schema(description = "DevCycle SDK Version")
-    val sdkVersion: String = BuildConfig.VERSION_NAME,
```

Comment:
This is the problem place. 

This is forever null now. Idk how to pull the name from the config.